README.md

```diff
@@ -1,6 +1,84 @@
 # WAM_APP_FRED
+
+WAM_APP_FRED is a Django App which is part of the [WAM project](https://github.com/rl-institut/WAM).
+
+
+## Overview
+
+This App is based on Django. In Django we have one project with n applications. 
```

Comment:
here it would make more sense to describe what purpose this app fulfills than to detail the WAM core structure